### DISCUSSION TITLE
Zusatzfelder in der App

### DISCUSSION THREAD

**Discussion by FrankAB1** [kimai/kimai#3575] [General]
Hallo,

ich habe Custom fields 1.23 installiert.
Wie bekomme ich Zusatzfelder hierzu in der jeweiligen App mit angezeigt?

Aktuell habe ich bei der Zeiterfassung An- & Abfahrt mit km Angabe eingebaut.
Funktioniert auch wunderbar wenn man direkt via Web zugreift.
Sollte aber in der App dann auch sichtbar sein.

Es sind beide App (Android & iOS) im Einsatz.
Über Infos hierüber würde ich mich freuen

**Comment by kevinpapst** [comment]
Hallo & Danke für die Nutzung von Kimai!
Die Apps sind von anderen Entwicklern als Kimai selber (letzteres bin quasi ich). 

Ich habe in der Vergangenheit extra Schnittstellen für die Zusatzfelder in die API eingebaut, grundsätzlich stehen die Infos auch den Apps zur Verfügung. Inwiefern diese dann genutzt werden, kann ich aber nicht sagen, habe ich selber nie ausprobiert... sollte ich wohl mal machen.

Jedenfalls sind im Marketplace die Entwickler auf den Detailseiten verlinkt, einfach mal an die wenden und nachfragen. 
Hier gibt es "nur" Support für Kimai selber.

**Comment by FrankAB1** [comment]
Hallo Kevin,

Danke für die schnelle Antwort.
Ich werde mich mal direkt an die Appentwickler wenden.
